ITE/6/README.adoc

```diff
@@ -141,38 +159,60 @@ The two main goals of this ITE are to:
     set of attestations that can be consumed by anyone. Furthermore, this paves
     the way for in-toto to support existing formats such as SPDX or CycloneDX.
 
-Furthermore, this ITE fits within the
-link:https://slsa.dev[SLSA framework]. The provenance format in particular is
-the official SLSA recommendation.
-
 [[reasoning]]
 == Reasoning
 
-See <https://github.com/in-toto/attestation#reasoning>. That spec was designed
-specifically for this ITE.
+The in-toto Attestation Framework specification provides more details and can be
+found
+link:https://github.com/in-toto/attestation/blob/main/docs/README.md#reasoning[here].
 
 [[backwards-compatibility]]
 == Backwards Compatibility
 
-New-style link-type attestations will be supported as a drop-in replacement for
-old-style links.
+The new framework supports a drop-in replacement for in-toto link metadata,
+meaning that the types of claims supported so far can continue to be made. Link
+predicates will be supported as a drop-in replacement for old-style links in
+in-toto's workflows.
 
-Old-style links will continue to be supported.
+Separately, in-toto implementations should continue to support old-style link
+metadata alongside attestations for a transitionary period of time. Implementers
+are free to determine the specific duration of this period, but it MUST be
+communicated to the in-toto community. For example, in-toto's reference
+implementation's timeline of support for old-style links will be discussed in
+the official roadmap.
 
 [[security]]
 == Security
 
-This does not affect security because the link-type attestation is isomorphic to
-the existing link schema and can be translated freely in both directions.
+This ITE does not affect the security of in-toto because:
+*   the link attestation type is isomorphic to the existing link schema and can be
+    translated freely in both directions.
+*   security must be evaluated in the context of each individual attestation
+    type, which is out of scope of this ITE.
```

Comment:
These bullets aren't rendering correctly. I think you need a blank line between the paragraph and the list?